moveos/moveos-stdlib/moveos-stdlib/sources/account_storage.move

```diff
@@ -5,30 +5,39 @@ module moveos_std::account_storage {
 
     use std::string::String;
     use std::signer;
+    use std::bcs;
     use moveos_std::type_table::{Self, TypeTable};
     use moveos_std::table::{Self, Table};
     use moveos_std::object::{Self, Object};
     use moveos_std::object_storage::{Self, ObjectStorage};
     use moveos_std::storage_context::{Self, StorageContext};
+    use moveos_std::tx_context;
 
     /// The account with the given address already exists
     const EAccountAlreadyExists: u64 = 0;
 
     /// The resource with the given type already exists
     const EResourceAlreadyExists: u64 = 1;
 
+    const NamedTableResource: u64 = 0;
+    const NamedTableModule: u64 = 1;
+
     struct AccountStorage has key {
         resources: TypeTable,
         modules: Table<String, vector<u8>>,
     }
 
+    //Ensure the NamedTableID generate use same method with Rust code
+    fun named_table_id(account: address, table_type: u64): address{
```

Comment:
Is the account storage contract transparent to application developers? The same as same table_type.

Reply:
yes, it is transparent